lms/djangoapps/courseware/views/views.py

```diff
@@ -898,12 +899,26 @@ def program_marketing(request, program_uuid):
     if not program_data:
         raise Http404
 
+    course_ids = get_course_ids(program_data['courses'])
+    total_courses_count = len(course_ids)
+
+    started_courses_count = CourseEnrollment.enrollments_for_user(request.user).filter(course_id__in=course_ids).count()
+
     program = ProgramMarketingDataExtender(program_data, request.user).extend()
     program['type_slug'] = slugify(program['type'])
     skus = program.get('skus')
     ecommerce_service = EcommerceService()
 
-    context = {'program': program}
+    price = program.get('price', '0.00')
+    if price != '0.00' and started_courses_count:
+        program['full_program_price'] = (float(price) / total_courses_count) * (total_courses_count - started_courses_count)
```

Comment:
It is possible to run into ZeroDivisionError in case of the empty program.
Please add some restriction or change the logic.

Reply:
@cmltaWt0 this code is not run unless there's at least one running course, if there're no courses then no no running courses as well. Is that what you've meant?

Reply:
@drtwisted Can't see this. Or it is not obvious. 
Please point me where this code runs only when course count > 0.
Currently, I can see that `get_course_ids(program_data['courses'])` can return on any number (event 0).